ddtrace/settings/config.py

```diff
@@ -334,7 +335,7 @@ class Config(object):
     available and can be updated by users.
     """
 
-    _extra_services_queue = None if in_aws_lambda() else get_mp_context().Queue(512)
+    _extra_services_queue = None if in_aws_lambda() or gevent_is_patched() else get_mp_context().Queue(512)
```

Comment:
Maybe we can just add if remote_config is disabled instead of gevent is patched ?

Reply:
It would make this PR a one line PR 😄